### PR TITLE
Sentry alerts review

### PR DESCRIPTION
**Pull request by MiguelAHM**
Archived alerts, not send clientError to sentry, and logg as clientError with info

### CODE REVIEW

**Comment by MiguelAHM** [comment]
Test ok -> https://gitlab.ripe.net/swe-database-team/whois/-/pipelines/394775

**Comment by MiguelAHM** [comment]
Test ok -> https://gitlab.ripe.net/swe-database-team/whois/-/pipelines/394775

File: whois-api/src/main/java/net/ripe/db/whois/api/httpserver/WhoisDoSFilter.java [hunk 1]
```diff
@@ -30,22 +31,25 @@ public class WhoisDoSFilter extends DoSFilter {
...
-                    return true;
+            } else {
+                final Ipv6Resource address = Ipv6Resource.parse(candidate);
+                for (Ipv6Resource entry : ipv6whitelist) {
+                    if (entry.contains(address)) {
+                        return true;
+                    }
                 }
             }
+        } catch (IllegalArgumentException ex){
```
Comment (MiguelAHM) [inline whois-api/src/main/java/net/ripe/db/whois/api/httpserver/WhoisDoSFilter.java:50]: put a comment inside the catch to specify parse it is failing

Comment (MiguelAHM) [reply whois-api/src/main/java/net/ripe/db/whois/api/httpserver/WhoisDoSFilter.java:50]: Check login, make sure exception is logged properly

Comment (MiguelAHM) [reply whois-api/src/main/java/net/ripe/db/whois/api/httpserver/WhoisDoSFilter.java:50]: Remove this code change

File: whois-scheduler/src/main/java/net/ripe/db/whois/scheduler/task/grs/ArinGrsSource.java [hunk 2]
```diff
@@ -102,9 +105,16 @@ private List<RpslObject> expand(final List<RpslAttribute> attributes) {
...
-                                final int begin = Integer.parseInt(rangeMatcher.group(1));
-                                final int end = Integer.parseInt(rangeMatcher.group(2));
+                                int begin;
+                                int end;
+                                try {
+                                    begin = Integer.parseInt(rangeMatcher.group(1));
+                                    end = Integer.parseInt(rangeMatcher.group(2));
+                                } catch (NumberFormatException ex) {
+                                    //Ignoring those asnumbers from Arin with big numbers
+                                    LOGGER.info(asnumber + " is too large to parse to Integer");
```
Comment (MiguelAHM) [inline whois-scheduler/src/main/java/net/ripe/db/whois/scheduler/task/grs/ArinGrsSource.java:115]: is larger than 32 bit limit in RIPE Database

Comment (MiguelAHM) [reply whois-scheduler/src/main/java/net/ripe/db/whois/scheduler/task/grs/ArinGrsSource.java:115]: curly bracers for logger variable

**Review by eshryane — APPROVED** [review]
Approve pending comments

**Pull request merged**